x-pack/test/api_integration/apis/metrics_ui/snapshot.ts

```diff
@@ -248,7 +248,7 @@ export default function ({ getService }: FtrProviderContext) {
           name: 'cpu',
           value: null,
           max: 0.47105555555555556,
-          avg: 0.0672936507936508,
+          avg: 0.47105555555555556,
```

Comment:
This is what the query returns:

```ts
[{ timestamp: 1562786355035, metric_0: null },
{ timestamp: 1562786415035, metric_0: null },
{ timestamp: 1562786475035, metric_0: null },
{ timestamp: 1562786535035, metric_0: null },
{ timestamp: 1562786595035, metric_0: null },
{ timestamp: 1562786655035, metric_0: 0.47105555555555556 },
{ timestamp: 1562786715035, metric_0: null }]
```

So, if we're filtering out null values from the average calculation, it makes sense for the expected result to be `0.47105555555555556`